script/core.hbs

```diff
@@ -86,3 +86,7 @@
   locale=(findFirst params.sdkLocaleOverride global_config.locale 'en')
   sdkVersion=global_config.sdkVersion
 }}
+
+{{#ifeq pageTemplate 'vertical-full-page-map'}}
```

Comment:
What do you all think about putting this logic here (rather than a page specific scripts partial)?

Reply:
I like being able to see all the scripts tags in one file, but it's also a little annoying for core.hbs to have to "know" the page template and manage that, when we can just put that in the page.html.hbs. Maybe we can come up with some kind of solution in a different pr?

Reply:
Now that we're using DOMContentLoaded, I think I can move the script back to the page template. Does that sound good for this PR?

Reply:
yeah sgtm! I think I like the idea of centralizing the scripts but not sure how we'd want to do it